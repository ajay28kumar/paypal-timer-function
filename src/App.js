import { makeStyles } from "@material-ui/core/styles";
import TimerContainer from "./components/timerContainer";
import './styles.css';

const useStyles = makeStyles({
  root: {
    justifyContent: "center",
    width: 960,
    ['@media (max-width:900px)']: { // eslint-disable-line no-useless-computed-key
      width: '100%'
    },
    margin: '0 32px',
    backgroundColor: '#fff',
    minHeight: '100vh'
  },
});

export default function App() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <TimerContainer/>
    </div>
  );
}
