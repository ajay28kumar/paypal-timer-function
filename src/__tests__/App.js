import { shallow } from "enzyme";
import App from "../App";

describe("Render App", () => {
  const app = shallow(<App />);
  it("snapshot testing", () => {
    expect(app).toMatchSnapshot();
  });

  it("render TimerContainer", () => {
    expect(app.find("TimerContainer").exists()).toBe(true);
  });
});
