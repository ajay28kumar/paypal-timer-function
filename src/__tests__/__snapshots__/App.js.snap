// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App snapshot testing 1`] = `ShallowWrapper {}`;
